addons/stock/static/src/views/list/stock_add_package_list_view.js

```diff
@@ -0,0 +1,35 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { listView } from "@web/views/list/list_view";
+import { ListController } from "@web/views/list/list_controller";
+import { useService } from "@web/core/utils/hooks";
+
+
+export class AddPackageListController extends ListController {
+    setup() {
+        super.setup();
+        this.actionService = useService("action");
+        this.pickingId = this.props.context.picking_id || 0;
+        this.packageIds = this.props.context.all_package_ids || [];
+    }
+
+    async onClickAdd(ev){
+        this.actionService.doAction({
+            type: "ir.actions.act_window",
+            res_model: "stock.add.entire.packages",
+            name: _t("Add a Package"),
+            views: [[false, "form"]],
+            target: "new",
+            context: {
+                "default_picking_id": this.pickingId,
+                "default_linked_package_ids": this.packageIds,
+            }
+        });
+    }
+}
+
+registry.category('views').add('stock_add_package_list_view', {
+    ...listView,
```

Comment:
De we really need a wizard for that ? why not "add a line" and fill the package column

Reply:
Because we're in a package list view. `Add a line` in that case would create a new package record, not add a package to that view.